src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -251,5 +233,76 @@ canvas {
 }
 
 .asset-error {
+    font-size: 85%;
+    font-weight: 500;
     margin: auto;
 }
+
+.badge-visited, .badge-tagged {
+    color: #fff;
+    position: absolute;
+    z-index: 2;
+    right: 0.5em;
+    top: 0.5em;
+    text-shadow: 0px 0px 4px #000
+}
+
+.badge-tagged {
+    background-color: rgba(green, 0.9);
+    border: 1px solid $lighter-2;
+}
```

Comment:
remove border for uniformity with other icons

Reply:
most of other icons are using in a toolbar and I would agree with that.  This border is on the badge, not the icon and feel like it needs a little separation from the image since its floating on top.  Its a very faint border only slightly lighter than the badge color itself.